js/gateio.js

```diff
@@ -2926,10 +2946,10 @@ module.exports = class gateio extends Exchange {
         const type = this.safeString (params, 'type', defaultType);
         const method = this.getSupportedMapping (type, {
             'swap': 'privateFuturesGetSettlePositions',
-            'futures': 'privateDeliveryGetSettlePositions',
+            'future': 'privateDeliveryGetSettlePositions',
         });
         const defaultSettle = type === 'swap' ? 'usdt' : 'btc';
-        const settle = this.safeString (params, 'settle', defaultSettle);
+        const settle = this.safeString (params, 'settle', defaultSettle).toLowerCase ();
```

Comment:
Let's use `this.safeStringLower` here and remove `toLowerCase()`.

Reply:
P.S. Everywhere throughout this file.

Reply:
There's one instance is `transfer` where `safeStringLower` wont work

Reply:
> There's one instance is transfer where safeStringLower wont work

Why not?

Reply:
I don't think we need to lowercase it on each call to `transfer`. Instead we should just use the `id` if it's already lowercase. If not, we should add `lowercaseId` alongside `id` in fetchCurrencies once instead of lowercasing it on every call.